it/mail/chrome/messenger/accountCreation.properties

```diff
@@ -1,73 +1,73 @@
-found_settings_guess=%1$S has found the settings for your email account.
+found_settings_guess=%1$S ha rilevato le impostazioni del tuo account e-mail.
 # LOCALIZATION NOTE(failed_to_find_settings): %1$S will be the brandShortName.
-failed_to_find_settings=%1$S non riesce a rilevare le impostazioni per l'account di posta.
+failed_to_find_settings=%1$S non riesce a rilevare le impostazioni del tuo account e-mail.
 manually_edit_config=Modifica configurazione
 
 # config subtitles
-check_preconfig=controllo configurazione parziale
-found_preconfig=Trovata una configurazione parziale
-checking_config=Controllo configurazione…
-found_config=Trovata una configurazione per questo account
-checking_mozilla_config=Controllo configurazioni della Comunità Mozilla…
-found_isp_config=Trovata una configurazione
-probing_config=verifica configurazione…
-guessing_from_email=provo ad indovinare la configurazione…
-config_details_found=Sono stati trovati i dettagli di configurazione!
+check_preconfig=controllo pre-configurazione in corso...
+found_preconfig=pre-configurazione rilevata
+checking_config=controllo configurazione in corso...
+found_config=rilevata configurazione per il tuo account
+checking_mozilla_config=controllo configurazioni community Mozilla in corso…
+found_isp_config=rilevata una configurazione
+probing_config=verifica configurazione in corso...
+guessing_from_email=ipotesi configurazione in corso…
+config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
 config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
-incoming_found_specify_outgoing=Le impostazioni per il server in entrata sono state rilevate, specificare il nome del server in uscita.
-outgoing_found_specify_incoming=Le impostazioni per il server in uscita sono state rilevate, specificare il nome del server in entrata.
-please_enter_missing_hostnames=Impossibile configurare automaticamente le impostazioni - fornire i nomi dei server mancanti.
-incoming_failed_trying_outgoing=Impossibile configurare automaticamente il server in ingresso, continuo col server in uscita.
-outgoing_failed_trying_incoming=Impossibile configurare automaticamente il server in uscita, continuo col server in entrata.
-checking_password=Controllo password…
+incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
+outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
+please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.
```

Comment:
CORRECTIONS NEEDED. It's better to avoid elision and apocopation when not necessary. Please modify the sentence as follows: 

 +please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.